Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKER_VERSION := 1
```

Comment:
no need to bump this, right?  we've moved to a new major version, so the Docker build resets to 1